source/Account.h

```diff
@@ -72,13 +72,18 @@ class Account {
 	// mortgages if a blank string is provided.
 	int64_t TotalDebt(const std::string &type = "") const;
 
+	// Get how many credits the player paid.
+	int64_t TotalPreviousPayment() const;
+
 
 private:
 	int64_t YearlyRevenue() const;
 
 
 private:
 	int64_t credits = 0;
+	// Keep track of how much was paid by the player.
+	int64_t totalPreviousPayment = 0;
```

Comment:
The getter function is beneath CreditScore, so the variable would make more sense beneath the creditScore variable .

Reply:
Fine by me
Related: next time someone touches mortgages they should poke that order